types/sic-ecies/index.d.ts

```diff
@@ -0,0 +1,98 @@
+// Type definitions for sic-ecies 1.0
+// Project: sic-ecies
+// Definitions by: Methrat0n <https://github.com/Methrat0n>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.2
+
+import { PrivateKey, PublicKey } from 'bitcore-lib';
+
+export interface ECIESOptions {
+  shortTag?: boolean;
+  noKey?: boolean;
+}
+
+export interface ECIES {
+  (opts?: ECIESOptions): ECIES;
+  privateKey: (privateKey: PrivateKey) => ECIES;
+  publicKey: (publicKey: PublicKey) => ECIES;
+  encrypt: (message: string | Buffer, ivbuf?: Buffer) => Buffer;
+  decrypt: (encbuf: Buffer) => Buffer;
+  Rbuf: Buffer;
+  kEkM: Buffer;
+  kE: Buffer;
+  kM: Buffer;
+}
+
+export interface AESCBC {
+  (): AESCBC;
+  encrypt: (messagebuf: Buffer, passwordstr: string) => Buffer;
+  decrypt: (encbuf: Buffer, passwordstr: string) => Buffer;
+  encryptCipherkey: (messagebuf: Buffer, cipherkeybuf: Buffer, ivbuf: Buffer) => Buffer;
+  decryptCipherkey: (encbuf: Buffer, cipherkeybuf: Buffer) => Buffer;
+}
+
+interface Blockcipher {
```

Comment:
Do you explicitly want this to not be exported? If it doesn't matter, then I would suggest removing `export {}` on line 98.

Reply:
Well, I choose this name myself without it being used in the library so I would prefer it to not be importable as it wasn't designed by the library maintainer.